packages/commcare/test/index.js

```diff
@@ -102,18 +101,17 @@ describe('getCases', () => {
   it('should fetch cases', async () => {
```

Comment:
I don't expect this test to change in this PR. We shouldn't be adding limits and offsets.